Dockerfile

```diff
@@ -10,7 +10,7 @@ RUN curl -L -o /tmp/jdk.tar.gz https://github.com/AdoptOpenJDK/openjdk8-binaries
 ENV JAVA_HOME /opt/java/openjdk
 
 # Install Node
-ENV NODE_VERSION 10.15.3
+ENV NODE_VERSION 13.10.1
```

Comment:
the latest long term support for Node.js from their [release page](https://nodejs.org/en/about/releases/) is currently`12.16.2` so we should use that instead.